docs/pages/reference/backends.mdx

```diff
@@ -172,6 +172,143 @@ teleport:
      etcd_max_client_msg_size_bytes: 15728640
 ```
 
+## PostgreSQL (Preview)
+
+<Admonition type="warning" title="Preview">
+  The PostgreSQL cluster state and audit log storage is currently in Preview.
```

Comment:
Should we also note that it's only available in 13.3+?

Reply:
Added a version warning in cdc9b599e7ed9646c8139890fdf884cd4cd7ed59